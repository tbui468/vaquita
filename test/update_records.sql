if exists drop database sol;
create database sol;
open sol;

create table planets (
                        name string, 
                        mass float, 
                        number_of_moons int,
                        surface_pressure float,
                        ring_system bool
                    );

insert into planets (
                        name, 
                        mass, 
                        number_of_moons, 
                        surface_pressure,
                        ring_system
                    ) 

values
        ("Venus", 4.87, 0, 92.0, false),
        ("Earth", 5.97, 1, 1.0, false),
        ("Mars", 0.642, 2, 0.01, false);

update planets set ring_system = true where number_of_moons = 2;
update planets set name = "Earth 2" where name = "Earth";
update planets set mass = 2.0, surface_pressure = 3.0, name = "Venus 2" where id = 1;

select * from planets;
close sol;
drop database sol;
exit;
