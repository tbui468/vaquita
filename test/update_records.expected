created database sol
opened database sol
created table planets
inserted 3 record(s) into planets
1, Venus 2, 2.000000, 0, 3.000000, false, 
2, Earth 2, 5.970000, 1, 1.000000, false, 
3, Mars, 0.642000, 2, 0.010000, true, 
closed database sol
dropped database sol
